aries-site/src/pages/whats-new.mdx

```diff
@@ -51,6 +51,17 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 --><!-- END: New Entry Template -->
 
 <!-- BEGIN: What's New Entries -->
+## August 30 - September 3, 2021
+
+- Published [Notifications page](/components/notification).
+- Published [Toast Notifications](/templates/toast-notifications).
+along with an example.
```

Comment:
```suggestion
- Published [Toast Notifications](/templates/toast-notifications).
```

Reply:
Let me know when either the "extra period" or dangling sentence is removed. I will approve and merge after that.